### DISCUSSION TITLE
add flutter as client

### DISCUSSION THREAD

**Discussion by Saeeed-B** [meteor/meteor#11963] [Feature Requests]
In my opinion, Flutter, with its great progress, is a good option to replace `cordova`.
It's amazing, we're getting rid of `WebView`
In addition, packages have been made

**Comment by harryadel** [comment]
This is already part of the [roadmap](https://github.com/meteor/meteor/blob/devel/docs/source/roadmap.md#next-major-releases)
>Explore Flutter as a front-end for Meteor apps;
